context/v1/EU/unstable.json

```diff
@@ -2,56 +2,49 @@
     "@context": {
         "@version": 1.1,
         "@protected": true,
-
         "id": "@id",
         "type": "@type",
-
         "description": "http://schema.org/description",
         "identifier": "http://schema.org/identifier",
+        "name": "http://schema.org/name",
+        "image": "http://schema.org/image",
         
-        
-
         "VaccinationCertificate": {
             "@id": "https://w3id.org/vaccination#VaccinationCertificate",
  
             "@context": {
                 "@version": 1.1,
                 "@protected": true,
-
                 "id": "@id",
                 "type": "@type",
-
                 "description": "http://schema.org/description",
                 "identifier": "http://schema.org/identifier",
-                "name": "http://schema.org/name"
+                "name": "http://schema.org/name",
+                "image": "http://schema.org/image"
             }
         },
 
         "VaccinationEvent": {
             "@id": "https://w3id.org/vaccination#VaccinationEvent",
-            
-
             "@context": {
                 "@version": 1.1,
                 "@protected": true,
-
                 "id": "@id",
                 "type": "@type",
-
                 "administeringCentre": {
-                    "@id": "http://hl7.org/fhir/Immunization.location.name",
+                    "@id": "http://hl7.org/fhir/uv/ips/Immunization.location.name",
```

Comment:
any chance we can make these terms resolvable?

Reply:
This is a bit hard to resolve at the moment, due to the following reasons:
 

1. Our first pass implementation had "resolvable" uri's such as `https://www.hl7.org/fhir/patient-definitions.html#Immunization.location`. These are "resolvable" in that it serves the purpose of having a human readable definition. @tplooker felt that having a _*.html_ extension was not appropriate. Also, this would not work with national or regional level customizations ("profiles") of FHIR, as in the above example you have pointed out: Organizations in charge of FHIR "Implementation Guides" for region or jurisdiction specific FHIR customizations publish "canonical" url's that are not resolvable, unfortunately. Hopefully this practice will change as jsonld adoption spreads (the next version of FHIR, R5 will support first class jsonld).
2. We could leverage the upcoming FHIR JSONLD work aimed at supporting full jsonld to rdf/ttl round tripping of FHIR resource model (it is not round-trippable today). However this won't be stable until it goes through the HL7 governance and balloting process for R5, as we understand it. E.g. [FHIRcat jsonld "base" definition of immunization](https://fhircat.org/fhir-r5/original/contexts/immunization.context.jsonld)

However, this is not as bad as it looks: as long a we stick to "canonical" URI's from the published FHIR registry (for national profiles), the intent of this vocabulary mapping (between w3c-ccg vax cert vocab and jurisdictional FHIR profile) can be unambiguous, from a machine level processing perspective. Just not human readable. 

On the durability of URI's: we could point to public FHIR registry end-points, some are run by private entities, for e.g.
[CA baseline profile for Immunization.location you have flagged](https://simplifier.net/canadianfhirbaselineprofilesca-core/locationprofile). Again, the canonical url for this profile is not resolvable, but most FHIR based applications know what to do with such conformance resource definitions (where to find them, enforce validations etc).